docker/cygnus-twitter/Dockerfile

```diff
@@ -46,7 +46,7 @@ ENV MVN_TGZ "apache-maven-${MVN_VER}-bin.tar.gz"
 ENV MVN_URL "http://www.eu.apache.org/dist/maven/maven-3/${MVN_VER}/binaries/${MVN_TGZ}"
 ENV MVN_HOME "/opt/apache-maven"
 
-ENV FLUME_VER "1.9.0"
+ENV FLUME_VER "1.11.0"
```

Comment:
Maybe it would be wise to include a FIXME like this in this file (and cygnus-ngsi-ld and cygnus-commons)

```suggestion
# FIXME: appart from upgrading Flume version in PR #2212, this Dockerfile has not been actually
# tested. It should be reviews and aligned with the one in cygnus-ngsi (for instance, we shouldn't need
# to directly download the flume package, as it should be installed through mvn based in the pom.xml
# configuration)
ENV FLUME_VER "1.11.0"
```